app/controllers/access_limit_controller.rb

```diff
@@ -3,25 +3,26 @@
 class AccessLimitController < ApplicationController
   rescue_from GdsApi::BaseError do |e|
     GovukError.notify(e)
-    render "#{action_name}_api_down", status: :service_unavailable
+    render "edit_api_down", status: :service_unavailable
   end
 
   def edit
     @edition = Edition.find_current(document: params[:document])
+    assert_edition_state(@edition, &:editable?)
   end
 
   def update
     result = AccessLimit::UpdateInteractor.call(params: params, user: current_user)
-    issues, edition = result.to_h.values_at(:issues, :edition)
+    @edition, issues = result.to_h.values_at(:edition, :issues)
```

Comment:
Why has the `edition` here been converted into an instance variable instead of passing to the partial via the assigns method?

Reply:
This is to fix a bug that the API down rescue would error because it expects an edition ivar.

Reply:
Ah okay 👍